StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -217,6 +224,7 @@ class EmbeddedPaymentMethodsView: UIView {
     }
 
     func updateSavedPaymentMethodRow(_ savedPaymentMethod: STPPaymentMethod?,
+                                     _ savedPaymentMethods: [STPPaymentMethod],
```

Comment:
Seems we can remove `_ savedPaymentMethod: STPPaymentMethod?,` as an argument?

Reply:
yeah probably can